docs/toolkit/ansible-avalanche-collection/tutorials/subnet-explorer.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 8
+---
+
+# Subnet Block Explorer
```

Comment:
Shouldn't this be `Blockchain Explorer`?

Reply:
Btw same for the faucet

Reply:
We can name the sections "Blockchain Explorer" and "Faucet" if you prefer but I thinks it's better to advertise these as "Subnet Blockchain Explorer" and "Subnet Faucet" for SEO

Reply:
When I think about it the Faucet can target multiple blockchains so it is okay to keep `Subnet Faucet`. I prefer `Subnet Blockchain Explorer` yes

Reply:
Renamed file and section in the latest commit